doc/man3/SSL_get_ciphers.pod

```diff
@@ -29,16 +29,16 @@ is returned.
 SSL_CTX_get_ciphers() returns the stack of available SSL_CIPHERs for B<ctx>.
 
 SSL_get1_supported_ciphers() returns the stack of enabled SSL_CIPHERs for
-B<ssl>, sorted by preference.
+B<ssl> as would be sent in a ClientHello (that is, sorted by preference).
 The list depends on settings like the cipher list, the supported protocol
 versions, the security level, and the enabled signature algorithms.
 SRP and PSK ciphers are only enabled if the appropriate callbacks or settings
 have been applied.
-This is the list that will be sent by the client to the server.
-The list supported by the server might include more ciphers in case there is a
-hole in the list of supported protocols.
-The server will also not use ciphers from this list depending on the
-configured certificates and DH parameters.
+Since this generates the list that would be sent in a ClientHello, it may
+differ from the list of acceptable ciphers when B<ssl> is the server side
+of a connection, as can occur when there is a
+hole in the list of supported protocols or the configured certificates and
+DH parameters limit the usable ciphers.
```

Comment:
Being non-native English speaker, I find this unnecessary complex. Since sentence in place of three, that suggests stronger causality between client and server lists than there is, which implied subject [I mean what's subject in "as can occur"?]...